serenity-report-resources/src/main/resources/freemarker/default.ftl

```diff
@@ -332,24 +332,24 @@
                 <#macro write_step(step, step_number)>
                 <@step_details step=step step_number=step_number level=level/>
                 <#if step.isAGroup()>
-                <#if level == 1>
```

Comment:
Removed these level checks to ensure the collapsible table header/footer is generated for all group steps.